cmd/clusterctl/client/cluster/installer.go

```diff
@@ -320,7 +328,23 @@ func (i *providerInstaller) getProviderContract(ctx context.Context, providerIns
 
 	releaseSeries := latestMetadata.GetReleaseSeriesForVersion(currentVersion)
 	if releaseSeries == nil {
-		return "", errors.Errorf("invalid provider metadata: version %s for the provider %s does not match any release series", provider.Version, provider.InstanceName())
+		availableSeries := []string{}
```

Comment:
```suggestion
		availableSeries := make([]string, 0, len(latestMetadata.ReleaseSeries))
```

Reply:
Thank you! I applied the commit suggestion!